pytorch_blade/src/compiler/jit/torch/freeze_module.cpp

```diff
@@ -336,7 +336,7 @@ class AttributePropagator {
       }
     } else if (attr.isTuple()) {
       auto tuple = std::move(attr).toTuple();
-      std::vector<IValue>& elems = tuple->elements();
+      std::vector<IValue> elems = tuple->elements();
```

Comment:
Why this is changed to non-ref type?

Reply:
The return type of `tuple->elements()` has changed to `const &`.